### PR TITLE
Remove the extract repetition in the name of the TREE extractor package.

### PR DESCRIPTION
**Pull request by constraintAutomaton**
Simply renamed the folder so that the repetition of the word "extract" would not be present. Since the importing is relative to the node modules and not the directory, it seems like there is no other changes are needed. I have also increased the version of the actor, because, on the one hand, with this change the CI pass and, on the other hand, with the previous PR #107, I think it is adequate to do so.

### CODE REVIEW

**Comment by rubensworks** [comment]
Does `yarn install` work locally for you?

**Comment by constraintAutomaton** [comment]
> Does `yarn install` work locally for you?

yes, even after the cleaning.

**Comment by constraintAutomaton** [comment]
Increasing the version of the actor solved the problem, but I don't really understand why. But, anyway with the previous PR, I think it is adequate to make that change.

File: engines/query-sparql-link-traversal/package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "@comunica/actor-extract-links-predicates": "^0.1.1",
     "@comunica/actor-extract-links-quad-pattern": "^0.1.1",
     "@comunica/actor-extract-links-quad-pattern-query": "^0.1.1",
-    "@comunica/actor-extract-links-tree": "^0.1.1",
+    "@comunica/actor-extract-links-tree": "^0.1.2",
```
Comment (rubensworks) [inline engines/query-sparql-link-traversal/package.json:52]: We shouldn't bump the version manually, as that will mess up our lerna publish command when we do a new release.

Could you revert this change?
Perhaps the CI failure was just a caching issue?

**Pull request closed without merging**